main.js

```diff
@@ -130,8 +157,15 @@ const fetchHandler = async function (req) {
   const urlPath = url.pathname.split("/").slice(1);
   const userAgent = req.headers.get("User-Agent");
 
-  if (userAgent && userAgent.includes("spplice/2") && !urlPath[0]) {
-    return Response.json(await utils.spplice(["get"]));
+  // Handle Spplice calls
+  if (!userAgent || userAgent.includes("spplice/2")) {
+
+    const path = `${epochtal.file.spplice.repository}/${urlPath[0]}`;
+    if (!fs.existsSync(path) || !urlPath[0])
+      return Response.json(await utils.spplice(["get"]));
```

Comment:
This is suuuper nitpicky, but we try to not omit curly brackets in cases like these.

Reply:
will do in a second